conf/readme-parts/links.md

```diff
@@ -2,8 +2,8 @@
 
 * Scramjet, the company behind [Transform Hub](https://scramjet.org)
 * The [Scramjet Framework - functional reactive stream processing framework](https://framework.scramjet.org)
-* The [Transform Hub repo onm github](https://github.com/scramjetorg/transform-hub)
-* You can see the [Scramjet Transform Hub API docs here](https://github.com/scramjetorg/transform-hub/tree/release/0.10/docs/development-guide/stream-and-api.md)
-* You can see the [CLI documentation here](https://github.com/scramjetorg/transform-hub/tree/release/0.10/docs/development-guide/scramjet-interface-cli.md), but `si help` should also be quite effective.
-* Don't forget to `star` this repo if you like it, `subscribe` to releases and keep visiting us for new versions and updates.
+* The [Transform Hub repo on github](https://github.com/scramjetorg/transform-hub)
+* You can see the [Scramjet Transform Hub API docs here](https://github.com/scramjetorg/transform-hub/tree/release/0.12/docs/development-guide/stream-and-api.md)
+* You can see the [CLI documentation here](https://github.com/scramjetorg/transform-hub/tree/release/0.12/docs/interfaces/CLI-command-reference.md), but `si help` should also be quite effective.
```

Comment:
@a-tylenda @agne6ka  are we're leaving both CLI/README and the interfaces/CLI-command-reference.md?

Reply:
@alicja-gruzdz  @agne6ka  we can decide on this in PR [#142](https://github.com/scramjetorg/transform-hub/pull/142)

Reply:
in the meantime @a-tylenda and I decided that we will close the https://github.com/scramjetorg/transform-hub/pull/142 branch and make the CLI correction on this branch. As for the destiny of interfaces/CLI-command-reference.md it hasn't been decided yet. @agne6ka what ya think?